### PR TITLE
updating the documentation to support accio

### LINKED ISSUE #144: Update Installation for Accio support

### PR DESCRIPTION
**Pull request by justingluck93**
Fixes #144 

This pull request **[adds new content]**.

## Summary of Changes
Added documentation for Accio support

### CODE REVIEW

File: docs/Getting Started/Installation/index.md [hunk 1]
```diff
@@ -2,8 +2,32 @@
...
+
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file
+
+1. Open the Package.swift file 
+2. Add the follwing line the dependencies array of you Package. We suggest always using the latest release. 
+    Please see [Mainifest format](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md) to specify dependencies to a specific branch / version of SDL.
+
+        .package(url: "https://github.com/smartdevicelink/sdl_ios.git", .from: "6.2.3"),
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Change `.from` to `.upToNextMajor(from: "6.2.0")`

File: docs/Getting Started/Installation/index.md [hunk 2]
```diff
@@ -2,8 +2,32 @@
...
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
 
+### Accio
+
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file
+
+1. Open the Package.swift file 
+2. Add the follwing line the dependencies array of you Package. We suggest always using the latest release.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: `follwing` -> `following`

`you` -> `your`

`latest release` -> `latest release of the SDL library`

File: docs/Getting Started/Installation/index.md [hunk 3]
```diff
@@ -2,8 +2,32 @@
...
 
+### Accio
+
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file
+
+1. Open the Package.swift file 
+2. Add the follwing line the dependencies array of you Package. We suggest always using the latest release. 
+    Please see [Mainifest format](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md) to specify dependencies to a specific branch / version of SDL.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Move below the code line and wrap in

!!!NOTE
"Text"
!!!

File: docs/Getting Started/Installation/index.md (hunk 1 above)
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Wrap in swift code block tags

File: docs/Getting Started/Installation/index.md [hunk 4]
```diff
@@ -2,8 +2,32 @@
...
+
+        .package(url: "https://github.com/smartdevicelink/sdl_ios.git", .from: "6.2.3"),
+            
+3. Add the following to the dependencies array of the target.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Change to:

`Add "SmartDeviceLink" or "SmartDeviceLinkSwift" to your dependencies array in your target. Use "SmartDeviceLink" for Objective-C applications and "SmartDeviceLinkSwift" for Swift applications.`

File: docs/Getting Started/Installation/index.md [hunk 5]
```diff
@@ -2,8 +2,32 @@
...
+
+        "SmartDeviceLink"
+            
+!!! NOTE
+If creating a swift application please `"SmartDeviceLinkSwift"` instead.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Remove this note based on the above change

File: docs/Getting Started/Installation/index.md [hunk 6]
```diff
@@ -2,8 +2,32 @@
...
+!!!
+
+1. Install SDL SDK for iOS:
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: "Install the SDK by running `accio install` in the root folder of your project in Terminal.

File: docs/Getting Started/Installation/index.md [hunk 7]
```diff
@@ -2,8 +2,32 @@
...
+
+        "SmartDeviceLink"
+            
+!!! NOTE
+If creating a swift application please `"SmartDeviceLinkSwift"` instead.
+!!!
+
+1. Install SDL SDK for iOS: 
+
+        accio install
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Remove

File: docs/Getting Started/Installation/index.md [hunk 8]
```diff
@@ -2,8 +2,32 @@
...
 
 ## Install SDL SDK
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
 
+### Accio
+
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Add a period at the end

File: docs/Getting Started/Installation/index.md (hunk 6 above)
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: This should be a 4

File: docs/Getting Started/Installation/index.md [hunk 9]
```diff
@@ -2,8 +2,32 @@
 In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software development kit (SDK) must be installed in your app. The following steps will guide you through adding the SDL SDK to your workspace and configuring the environment.
 
 ## Install SDL SDK
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
 
+### Accio
+
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Remove this extra line

File: docs/Getting Started/Installation/index.md (hunk 1 above)
Comment (justingluck93) [reply docs/Getting Started/Installation/index.md]: changed

File: docs/Getting Started/Installation/index.md (hunk 2 above)
Comment (justingluck93) [reply docs/Getting Started/Installation/index.md]: changed

File: docs/Getting Started/Installation/index.md (hunk 1 above)
Comment (justingluck93) [reply docs/Getting Started/Installation/index.md]: I'm unsure what you mean by this. The code shows up formatted on the documentation page

Comment (joeljfischer) [reply docs/Getting Started/Installation/index.md]: We don't format code by using indentation. We format code using backticks. See other guides. This is technically swift code, and should be formatted as such using "```swift"

File: docs/Getting Started/Installation/index.md [hunk 10]
```diff
@@ -2,8 +2,26 @@
...
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
 
+### Accio
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file.
+
+1. Open the Package.swift file 
+2. Add the following line the dependencies array of you Package. We suggest always using the latest release of the SDL library.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]:
```suggestion
2. Add the following line to the dependencies array of your package file. We suggest always using the latest release of the SDL library. 
```

File: docs/Getting Started/Installation/index.md [hunk 11]
```diff
@@ -2,8 +2,26 @@
...
 ## Install SDL SDK
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
 
+### Accio
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file.
+
+1. Open the Package.swift file
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]:
```suggestion
1. Open the Package.swift file.
```

File: docs/Getting Started/Installation/index.md [hunk 12]
```diff
@@ -2,8 +2,26 @@
 In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software development kit (SDK) must be installed in your app. The following steps will guide you through adding the SDL SDK to your workspace and configuring the environment.
 
 ## Install SDL SDK
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
 
+### Accio
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]:
```suggestion
You can install this library using [Accio](https://github.com/JamitLabs/Accio), which is based on SwiftPM syntax. Please follow the steps on the Accio README linked above to initialize Accio into your application.
```

File: docs/Getting Started/Installation/index.md [hunk 13]
```diff
@@ -2,8 +2,26 @@
 In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software development kit (SDK) must be installed in your app. The following steps will guide you through adding the SDL SDK to your workspace and configuring the environment.
 
 ## Install SDL SDK
-There are three different ways to install the SDL SDK in your project: CocoaPods, Carthage, or manually.
+There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.
 
+### Accio
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]:
```suggestion
Once installed and initialized into your Xcode project, the root directory should contain a Package.swift file.
```

File: docs/Getting Started/Installation/index.md [hunk 14]
```diff
@@ -2,8 +2,26 @@
...
+
+1. Open the Package.swift file 
+2. Add the following line the dependencies array of you Package. We suggest always using the latest release of the SDL library. 
+
+        .package(url: "https://github.com/smartdevicelink/sdl_ios.git", .upToNextMajor(from: "6.2.0"),
+        
+!!! NOTE
+Please see [Mainifest format](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md) to specify dependencies to a specific branch / version of SDL.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]:
```suggestion
Please see [package manifest format](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md) to specify dependencies to a specific branch / version of SDL.
```

File: docs/Getting Started/Installation/index.md [hunk 15]
```diff
@@ -2,8 +2,26 @@
...
+!!!
+            
+3. Add `"SmartDeviceLink"` or `"SmartDeviceLinkSwift"` to your dependencies array in your target. Use `"SmartDeviceLink"`for Objective-C applications and `"SmartDeviceLinkSwift"` for Swift applications.
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]:
```suggestion
3. Add `"SmartDeviceLink"` or `"SmartDeviceLinkSwift"` to the dependencies array in your target. Use `"SmartDeviceLink"`for Objective-C applications and `"SmartDeviceLinkSwift"` for Swift applications.
```

File: docs/Getting Started/Installation/index.md [hunk 16]
```diff
@@ -2,8 +2,26 @@
...
+You can install this library using [Accio/SwiftPM](https://github.com/JamitLabs/Accio) documentation page.  Please follow the steps to install and initialization Accio into a current or new application. 
+
+Once installed and initialized into your xcode project the root directory should contain a Package.swift file.
+
+1. Open the Package.swift file 
+2. Add the following line the dependencies array of you Package. We suggest always using the latest release of the SDL library. 
+
+        .package(url: "https://github.com/smartdevicelink/sdl_ios.git", .upToNextMajor(from: "6.2.0"),
+        
+!!! NOTE
```
Comment (joeljfischer) [inline docs/Getting Started/Installation/index.md]: Try indenting the NOTE to keep the numbering working.

**Pull request closed without merging**